### ISSUE TITLE
Food changes

### ISSUE THREAD

**Issue by AndreiAvinov** [skyarkhangel/Hardcore-SK#2218]
Alright, weapons are _kinda_ fine for now. Now for food: rn there are like 3-4 types of meals that people make: cheese sandwiches (for nutrition efficiency), mushroom stews (for buffs), roasted meat (for simplicity). The choices are kinda logical and I want to expand on them. I thought about having 3 main food chains (bakery, mushrooms stuff and vanilla-like nutrition efficient food) and have a few specialized meals for professional pawns. Like fruit pie can give buffs to sight and some other soldier-related stats. Pizza will be the ultimate food that requires a lot of different ingredients but provide a wide variety of buffs (though not as strong as specialized meals). With the latest 1.1 patch animal husbandry is a lot more important and i want to give it even more meaning in the late game (like having a few cows for milk).
All that with a lot of tweaks to basic production and overall balance that you will hopefully like.

**Comment by sidfu** [comment]
food should be looked at by type.
baked
normal meals
quick meals
and then treats.

tbh we need to start at the crops and redo from ground up values  for it and how they grow. easy crops= low food harder crops give more food value and such.

**Comment by AndreiAvinov** [comment]
Ye, crops are all over the tree noww too. I will return to it some time later coz another person is trying to reorganize the tech tree and I will go from there then the time comes (it will be mostly numbers changes anyway)

**Comment by AndreiAvinov** [comment]
Ok, back to work - had a few redirections along the way, but I got a better idea of what is lacking.
First of all - fixes.
I have a few questions to sb who maybe worked on food before (and for you, @skyarkhangel )
- Why food has negative beauty? srsly why?
- Do we really need cuttable coconut trees? Can we just make them as normal fruit trees? This will give me an opportunity to balance them easier and will create a new layer of decision-making when playing in the desert - to cut trees or to let them grow for food.
- The whole fruit section is a bit broken with nutrition values and cost all over the place, fruit salad being the only meal that _decreases_ nutrition
Also, Sid, I thought about your system with preparing ingredients and it kinda feels unnecessary
Basically what I'm trying to do is to make what we have work and just a few logical shifts in crafting chains + maybe adding some missing food from VGP that really has its niche
